presto-main/src/main/java/io/prestosql/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -347,6 +347,13 @@ private boolean isColumnsEnumeratingTable(InformationSchemaTable table)
         return Optional.empty();
     }
 
+    private Set<String> removeEmptyValues(Set<String> values)
```

Comment:
static
this method is very much of utility kind, so I would define it at the very bottom of this class